.github/workflows/tx-pull.yml

```diff
@@ -0,0 +1,16 @@
+name: Get translations from transifex
+
+on:
+  schedule:
+    - cron: '17 5 * * 1,4'
```

Comment:
I set that for twice a week, but I didn't really had much idea about the update cadence we wanted. We should also keep in mind that if needed we can also trigger it manually

Reply:
maybe 1 a week would be a more sensible default?

Reply:
I think 1 week is fine. We can change it if the volume of translations is high.

Reply:
```suggestion
    - cron: '17 5 * * 1'
```